packages/example-app-base/src/components/FeedbackList/FeedbackList.tsx

```diff
@@ -0,0 +1,122 @@
+
+export interface IFeedbackListProps {
+  title: string;
+}
+
+export interface IFeedbackListState {
+  openIssues: IListItem[];
+  closedIssues: IListItem[];
+}
+
+export interface IListItem {
+  issueTitle: string;
+  issueNum: number;
+  issueCreated: string;
+}
+
+export class FeedbackList extends React.Component<IFeedbackListProps, IFeedbackListState, IListItem> {
+  constructor(props: IFeedbackListProps) {
+    super(props);
+    this.state = {
+      openIssues: [],
+      closedIssues: []
+    };
+  }
+
+  public async componentDidMount(): Promise<void> {
+    const githubUrl =
+      'https://api.github.com/search/issues?q=type:issue%20repo:OfficeDev/office-ui-fabric-react%20label:%22Component:%20' +
+      this.props.title;
+
+    const openIssuesURL = githubUrl + '%22%20is:open';
+    const closedIssuesURL = githubUrl + '%22%20is:closed';
+
+    const results = await Promise.all([this.getIssues(openIssuesURL), this.getIssues(closedIssuesURL)]);
+
+    this.setState({ openIssues: results[0], closedIssues: results[1] });
+  }
+
+  public getIssues(url: string): IListItem[] {
+    let issueList: IListItem[] = [];
+    fetch(url)
+      .then((response: Response) => {
+        return response.text();
+      })
+      .then((responseText: string) => {
+        const myObj = JSON.parse(responseText);
+        for (let i = 0; i < myObj.total_count; i++) {
+          let dateCreated = new Date(myObj.items[i].created_at);
+          let openedOn = relativeDates(dateCreated, new Date());
+          issueList.push({
+            issueTitle: myObj.items[i].title,
+            issueNum: myObj.items[i].number,
+            issueCreated: openedOn
+          });
+        }
+      });
+    return issueList;
+  }
+
+  public render(): JSX.Element | null {
+    let { openIssues, closedIssues } = this.state;
+
+    return (
+      <div>
+        <PrimaryButton
+          href="https://github.com/OfficeDev/office-ui-fabric-react/issues/new/choose"
+          target="_blank"
+          primary={true}
+          className="FeedbackList-button"
+        >
+          Submit GitHub Issue
+        </PrimaryButton>
+
+        <Pivot className="FeedbackList-pivot">
+          <PivotItem linkText="Open Issues">
+            <List
+              items={openIssues}
+              onRenderCell={this._onRenderCell}
+              data-is-scrollable={true}
+              className="FeedbackList-issueList"
+            />
+          </PivotItem>
+          <PivotItem linkText="Closed Issues">
+            <List
+              items={closedIssues}
+              onRenderCell={this._onRenderCell}
+              data-is-scrollable={true}
+              className="FeedbackList-issueList"
+            />
+          </PivotItem>
+        </Pivot>
+      </div>
+    );
+  }
+
+  private _onRenderCell(item: IListItem, index: number, isScrolling: boolean): JSX.Element {
+    return (
+      <div className="ms-ListGhostingExample-itemCell" data-is-focusable={true}>
```

Comment:
yes, they provide some of the styling for the list

Reply:
Okay, I just didn't see the styles added in this PR.

Reply:
I used this style for consistency because ExampleCard and ComponentPage within this same folder are doing it like that with the global classes. Should I still change it to local?